01_fib/baijie/fib.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+// fib function to generate fibonacci sequence
```

Comment:
This comment is a bit redundant - it's easy to see what the function does without it. If you feel that you must comment, a suggestion would be 
```suggestion
// fib generates a fibonacci sequence for _n_. It expects the existence of a 
// variable named _out_ of type _io.Writer_, and will output the sequence 
// to that variable; this can be used for eg. printing on screen or testing.
```

Personally I'd just leave it out for simplicity.

Reply:
Done